chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -246,6 +248,17 @@ def make_zone(zone)
   end
 end
 
+service "bind9" do
+  case node[:platform_family]
+  when "rhel", "suse"
+    service_name "named"
+  end
+  supports restart: true, status: true, reload: true
+  running true
+  enabled true
+  action :enable
+end
```

Comment:
Wrong rebase here: this service definition actually got moved, a few lines later, so you should not readd it, but update the moved one.